plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx

```diff
@@ -82,7 +82,7 @@ const FailSkippedWidget = ({
   return null;
 };
 
-const generatedColumns: TableColumn[] = [
+export const ciTableColumns: TableColumn<Project>[] = [
```

Comment:
Ah, exporting the existing columns expands the scope a bit, could you please use [this](https://github.com/backstage/backstage/blob/49e92a6f85066e7ddd653a62783effce420b082b/plugins/github-deployments/src/components/GithubDeploymentsTable/GithubDeploymentsTable.tsx#L68) pattern for exporting the columns? It's what we've settled on for all plugins. This pattern makes it a lot easier to re-use and potentially configure individual columns.

Reply:
I have made changes to the codebase to meet the set standards. Please let me know if you would like me to make any other changes to it again before we merge it.

@Rugvip  @NishkarshRaj